### PR TITLE
Updates to Book of Air and Book of Earth

### PR DESCRIPTION
**Pull request by simonakey**
Couple of typo fixes
Added page numbers to spells

### CODE REVIEW

File: packs/book_of_earth_pack/xml/boe_katas.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <Requirements field="crab_hida_bushi_school" type="tag">Hida Bushi School</Requirements>
       <Requirements field="crab_hiruma_scout_school" type="tag">Hiruma Scout School</Requirements>
       <Requirements field="phoenix_shiba_bushi_school" type="tag">Matsu Berserker School</Requirements>
-      <Requirements field="crane_daidoji_iron_warrior" type="tag">Daidoji Iron Warrior</Requirements>
+      <Requirements field="crane_daidoji_iron_warrior_school" type="tag">Daidoji Iron Warrior</Requirements>
```
Comment (OpenNingia) [inline packs/book_of_earth_pack/xml/boe_katas.xml:18]: This is not right, the Daidoji Iron Warrior's school id is actually 'crane_daidoji_iron_warriors' (check the core pack). It doesn't follow the "_school" convention probably by mistake, but we cannot change it (or we break all Daidoji Iron Warrior character saves)

File: packs/book_of_earth_pack/xml/boe_katas.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
       <Requirements field="crab_hida_bushi_school" type="tag">Hida Bushi School</Requirements>
       <Requirements field="crab_hida_pragmatist_school" type="tag">Hida Pragmatist School</Requirements>
       <Requirements field="phoenix_shiba_bushi_school" type="tag">Matsu Berserker School</Requirements>
-      <Requirements field="crane_daidoji_iron_warrior" type="tag">Daidoji Iron Warrior</Requirements>
+      <Requirements field="crane_daidoji_iron_warrior_school" type="tag">Daidoji Iron Warrior</Requirements>
```
Comment (OpenNingia) [inline packs/book_of_earth_pack/xml/boe_katas.xml:27]: Same issue with Daidoji Iron Warrior School

**Review by OpenNingia — CHANGES_REQUESTED** [review]

File: packs/book_of_earth_pack/xml/boe_katas.xml (hunk 2 above)
Comment (simonakey) [reply packs/book_of_earth_pack/xml/boe_katas.xml]: Reverted

File: packs/book_of_earth_pack/xml/boe_katas.xml (hunk 1 above)
Comment (simonakey) [reply packs/book_of_earth_pack/xml/boe_katas.xml]: Reverted

**Pull request merged**